src/database/database-files.cpp

```diff
@@ -27,355 +27,6 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "util/string.h"
 #include <cassert>
```

Comment:
Headers `"remoteplayer.h"`, `"settings.h"`, `"server/player_sao.h"`, and `<cassert>` are no longer needed.